.github/workflows/build.yml

```diff
@@ -156,30 +183,32 @@ jobs:
           mingw-w64-${{ matrix.config.arch }}-ntldd
     - name: Set Environment Variables
       run: |
+        bash --version
         echo "$HOME/.local/bin" >> "$GITHUB_PATH"
-        echo "INSTALL_REF=${GITHUB_REF##*/}" >> "$GITHUB_ENV"
         if [[ "${MSYSTEM}" == "MINGW64" ]]; then
-          echo "INSTALL_NAME=lite-xl-${GITHUB_REF##*/}-windows-x86_64" >> "$GITHUB_ENV"
+          echo "ARCH=x86_64-windows" >> "$GITHUB_ENV"
+          echo "ADDONS=--addons" >> "$GITHUB_ENV"
         else
-          echo "INSTALL_NAME=lite-xl-${GITHUB_REF##*/}-windows-i686" >> "$GITHUB_ENV"
+          echo "ARCH=x86-windows" >> "$GITHUB_ENV"
         fi
-    - name: Install Dependencies
-      if: false
-      run: bash scripts/install-dependencies.sh --debug
     - name: Build
-      run: |
-        bash --version
-        bash scripts/build.sh -U --debug --forcefallback
-    - name: Package
-      run: bash scripts/package.sh --version ${INSTALL_REF} --debug --binary
+      run: scripts/build.sh $ADDONS -U --debug --forcefallback
+    - name: Package Portable
+      run: cd build-$ARCH && zip ../lite-xl-${{ needs.version.outputs.ref }}-$ARCH.zip -r lite-xl && cd ..
```

Comment:
```suggestion
      run: |
        cd build-$ARCH
        zip ../lite-xl-${{ needs.version.outputs.ref }}-$ARCH.zip -r lite-xl
        cd ..
```

Reply:
I honestly find it clearer to have it one line; it's a "semantic packet" for lack of a better word; it's exactly one atomic action taken in the line.

Reply:
No, it makes the line longer than necessary. It all works the same since bash still stops if one of the commands errored, but laying them out in each lines makes it much easier to read.

Reply:
I disagree, but if you feel strongly about it; I'll change it.